src/main/webapp/user-page.html

```diff
@@ -75,11 +75,16 @@ <h1 class="site-heading text-center text-white d-none d-lg-block">
               <h2 class="section-heading mb-4">
                 <span class="section-heading-lower">About Me</span>
               </h2>
+              <div class="with-events text-center" id="nickname-container">
```

Comment:
Thanks JingQi! Maybe there can be something for users to distinguish the nickname input from the introduction input here?

Reply:
Okay! I think I use what Caleb suggested?